app/src/main.js

```diff
@@ -8,6 +9,7 @@ import router from './router'
 import store from './store'
 
 Vue.use(VueMaterial)
+Vue.use(vPlayBack)
```

Comment:
This seems like it could be a heavy dependency. You may want to lazy load it in the components where it's used instead of setting as a global component. This should put it into a separate bundle on build.

Reply:
Good idea, fixed in 5ddb79f.